appservice/intent.go

```diff
@@ -542,3 +542,7 @@ func (intent *IntentAPI) EnsureInvited(ctx context.Context, roomID id.RoomID, us
 	}
 	return nil
 }
+
+func (intent *IntentAPI) GetEvent(ctx context.Context, roomID id.RoomID, eventID id.EventID) (*event.Event, error) {
+	return intent.Client.GetEvent(ctx, roomID, eventID)
+}
```

Comment:
This is unnecessary, functions only need to be wrapped if there's additional special behavior (you can already call any Client functions on IntentAPI because it's an unnamed/embedded struct)